tensorflow/lite/micro/kernels/pad.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_LITE_MICRO_KERNELS_PAD_H_
+#define TENSORFLOW_LITE_MICRO_KERNELS_PAD_H_
+
+#include "third_party/tensorflow/lite/c/common.h"
+
+namespace tflite {
+namespace ops {
+namespace micro {
+namespace pad {
+
+TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node);
```

Comment:
nit: can you make this a flat tflite namepspace (http://b/170230530)?

The function would have to become `PadPrepare`, similar to https://github.com/tensorflow/tflite-micro/blob/b4279b999d34ed71a57f53ce47162f8a1f7f133b/tensorflow/lite/micro/kernels/conv.h#L77

Reply:
Done, cleaned up the xtensa version as well to keep them aligned.